modules/react/breadcrumbs/lib/Breadcrumbs.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import {createContainer} from '@workday/canvas-kit-react/common';
+import {Menu} from '@workday/canvas-kit-react/menu';
+import {useBreadcrumbsModel} from './hooks/useBreadcrumbsModel';
+import {BreadcrumbsList} from './BreadcrumbsList';
+import {BreadcrumbsOverflowButton} from './BreadcrumbsOverflowButton';
+import {BreadcrumbsItem} from './BreadcrumbsItem';
+import {BreadcrumbsLink} from './BreadcrumbsLink';
+import {BreadcrumbsCurrentItem} from './BreadcrumbsCurrentItem';
+import {BreadcrumbsMenu} from './BreadcrumbsMenu';
+
+export interface BreadcrumbsProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * The accessibility label for the nav element.
+   *
+   * @default "breadcrumbs"
+   */
+  'aria-label'?: string;
+  /**
+   * The contents of the Breadcrumbs. Can be `Breadcrumbs` children or any valid elements.
+   */
+  children: React.ReactNode;
+}
+
+export const Breadcrumbs = createContainer()({
+  displayName: 'Breadcrumbs',
+  modelHook: useBreadcrumbsModel,
+  subComponents: {
+    List: BreadcrumbsList,
+    Item: BreadcrumbsItem,
+    OverflowButton: BreadcrumbsOverflowButton,
```

Comment:
```suggestion
    /**
     * ### Breadcrumbs Overflow Button
     * ---
     * [View Developer Docs](https://canvas.workdaydesign.com/components/navigation/breadcrumbs/#breadcrumbsoverflowbutton)
     * 
     * The toggle button for the Breadcrumbs Menu.
     * This button is rendered implicitly in the `Breadcrumbs.List` when the list overflows.
     * However, if you need to pass props to it, you can do so by passing `overflowButtonProps` to the List.
     * 
     * @example
     * ```tsx
     * <Breadcrumbs.List
     *   overflowButtonProps={{
     *     'aria-label': 'More breadcrumb links',
     *     onClick={handleOverflowButtonClick}
     *   }}
     * >
     * ```
     */
    OverflowButton: BreadcrumbsOverflowButton,
```